### PR TITLE
nixpkgs doc: add installCheck phase

### PR DESCRIPTION
**Pull request by ericsagnes**
###### Motivation for this change

Document the installCheck phase.
###### Things done
- Built on platform(s)
  - [x] NixOS
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by ericsagnes** [comment]
cc @edolstra @zimbatm

File: doc/stdenv.xml [hunk 1]
```diff
@@ -988,6 +988,42 @@ set debug-file-directory ~/.nix-profile/lib/debug
 
 </section>
 
+<section xml:id="ssec-installCheck-phase"><title>The installCheck phase</title>
+
+<para>The installCheck phase checks whether the the package was installed
```
Comment (FRidh) [inline doc/stdenv.xml]: the the

File: doc/stdenv.xml [hunk 2]
```diff
@@ -988,6 +988,42 @@ set debug-file-directory ~/.nix-profile/lib/debug
...
+<variablelist>
+  <title>Variables controlling the installCheck phase</title>
+
+  <varlistentry>
+    <term><varname>doInstallCheck</varname></term>
+    <listitem><para>If set to a non-empty string, the installCheck phase is
+    executed, otherwise it is skipped (default).  Thus you should set
+
+    <programlisting>
+doInstallCheck = true;</programlisting>
```
Comment (FRidh) [inline doc/stdenv.xml]: indentation?

**Review by FRidh — CHANGES_REQUESTED** [review]

File: doc/stdenv.xml (hunk 1 above)
Comment (ericsagnes) [inline doc/stdenv.xml]: Fixed, thanks!

File: doc/stdenv.xml (hunk 2 above)
Comment (ericsagnes) [inline doc/stdenv.xml]: Fixed, thanks!

**Comment by zimbatm** [comment]
Awesome, nice work!

**Comment by ericsagnes** [comment]
Thanks for merging!

**Pull request merged**